src/components/NavBar/MenuItem/index.tsx

```diff
@@ -12,7 +12,12 @@ import {
 } from '@material-ui/core';
 
 import { MenuItemType, LayerType, TableType } from '../../../config/types';
-import { addLayer, removeLayer } from '../../../context/mapStateSlice';
+import {
+  removeLayers,
```

Comment:
Let's try to keep an alphabetical order when possible.

For simplicity, maybe we can just keep the `multiple` version and use `removeLayers([layerId])` if we need to apply it to a single entity?